frontend/src/wasm-communication/messages.ts

```diff
@@ -107,6 +108,7 @@ export class FrontendNode {
 	readonly disabled!: boolean;
 
 	readonly thumbnailSvg!: string | undefined;
+	category: any;
```

Comment:
For consistency with the other entries here, this should probably be `readonly category!: string;`

Reply:
That doesn't do anything.